go/obscuronode/nodecommon/types.go

```diff
@@ -4,17 +4,21 @@ import (
 	"fmt"
 	"sync/atomic"
 
+	"github.com/ethereum/go-ethereum/core/types"
+
 	"github.com/obscuronet/obscuro-playground/go/hashing"
 	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
 
 	"github.com/ethereum/go-ethereum/common"
 )
 
 // Todo - this has to be a trie root eventually
```

Comment:
this todo should be on the StateRoot type

Reply:
Addressed.